cms/djangoapps/models/settings/course_metadata.py

```diff
@@ -260,7 +266,30 @@ def validate_and_update_from_json(cls, descriptor, jsondict, user, filter_tabs=T
                 'The proctoring provider cannot be modified after a course has started.'
                 ' Contact {support_email} for assistance'
             ).format(support_email=settings.PARTNER_SUPPORT_EMAIL or 'support')
-            errors.append({'message': message, 'model': filtered_dict.get('proctoring_provider')})
+            errors.append({'message': message, 'model': proctoring_provider_model})
+
+        # Require a valid escalation email if Proctortrack is chosen as the proctoring provider
+        escalation_email_model = filtered_dict.get('proctoring_escalation_email')
```

Comment:
In the case of the advanced settings page it would always exist.  However the CourseMetadata functions support partial updates so I've written it to cover that use case.